src/main/java/com/bgsoftware/superiorskyblock/commands/CmdWarp.java

```diff
@@ -64,6 +68,22 @@ public void execute(SuperiorSkyblockPlugin plugin, CommandSender sender, String[
             return;
 
         SuperiorPlayer superiorPlayer = plugin.getPlayers().getSuperiorPlayer(sender);
+
+        if (args.length == 3) {
```

Comment:
You assume the third parameter is always the warp's name, however the second parameter is optional.
What will happen if player will run /is warp <warp-name>? The logic you wrote will not occur, and if I am not mistaken, it should. The correct flow should be:
If there are 3 arguments -> third one is the warp's name
If there are two arguments -> check if the second argument is the warp's name.
If it's not the warp's name, then it's probably an island/player's name, and the old logic should occur regularly.
If it is a warp's name, then the player will be teleported to that warp.

Also, not all players can teleport to all warps, and there must be a check for that.
In the original code, the check is done in the requestObjects of MenuWarps.